saml.Test/saml.Test.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Also can you investigate why this might be happening (see warning arrows in picture below):

![image](https://user-images.githubusercontent.com/44148159/49914602-100caf00-fee6-11e8-81de-470880f8c396.png)

Does this happen for you if you do a clone of the repo to a fresh directory and then switch to your branch?